test/unit/discovery_rule_test.rb

```diff
@@ -9,6 +9,57 @@ class DiscoveryRuleTest < ActiveSupport::TestCase
     @hostgroup.save!
   end
 
+  context "#validates attribute" do
+    setup do
+      @huge_name = SecureRandom.hex(128)
```

Comment:
Thank you for the patch, this is not too obvious that this generates actually 256 characters, could you change it to something like `"X" * 256`?